### DISCUSSION TITLE
Is there a way to statically host the output with icons?

### DISCUSSION THREAD

**Discussion by MetalGooseSolid** [mudassir0909/jsonresume-theme-elegant#126] [Q&A]
I can use the jsonresume hosting option and I can run the server as per the instructions. I was just wondering if there was some way to statically compile the web content (with icons) to make it easy to selfhost? If this is supposed to be easy I completely missed how to do it.

**Comment by mudassir0909** [comment]
So, it uses icomoon which I believe generates a custom font file and generates CSS classes. Here's a wiki page on how to add/modify icons https://github.com/mudassir0909/jsonresume-theme-elegant/wiki/Adding-a-new-icon

About your question, as long as you grab the icons from https://github.com/mudassir0909/jsonresume-theme-elegant/tree/master/assets/icomoon/fonts and host yourself, it should be ok?
